static/app/utils/fields/index.ts

```diff
@@ -677,7 +677,6 @@ const EVENT_FIELD_DEFINITIONS: Record<AllEventFieldKeys, FieldDefinition> = {
   [FieldKey.LEVEL]: {
     kind: FieldKind.FIELD,
     desc: t('Severity of the event (i.e., fatal, error, warning)'),
-    keywords: ['fatal', 'critical', 'error', 'warning', 'log', 'info', 'debug'],
```

Comment:
Does it autocomplete them as tags then? how does it work

Reply:
yes it will do a regular tagstore search for them

Reply:
i'm also curious if they are actually set to anything else, but i don't see any ENUM enforcement, so just being careful